java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -219,7 +219,10 @@ public enum OrcConf {
       "orc.proleptic.gregorian.default", false,
       "This value controls whether pre-ORC 27 files are using the hybrid or proleptic\n" +
       "calendar. Only Hive 3.1 and the C++ library wrote using the proleptic, so hybrid\n" +
-      "is the default.")
+      "is the default."),
+  ROW_BATCH_SIZE("orc.row.batch.size", "orc.row.batch.size", 1024,
+          "The number of rows to include in a orc vectorized reader batch. " +
```

Comment:
My example included this line too.

Reply:
IntelliJ IDEA sometimes hinted that I didn't notice it was a few characters indent, I fixed it now, thanks again for your patience.

![image](https://user-images.githubusercontent.com/3898450/167280527-e25914c6-adba-4fbe-afd3-b56103a7fe52.png)